<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QMainWindow" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="messageList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="contactsList">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>120</y>
      <width>161</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="messageEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Digite aqui uma nova mensagem</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="communicationMeter">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>490</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="suffix">
     <string>km</string>
    </property>
    <property name="minimum">
     <number>5</number>
    </property>
    <property name="maximum">
     <number>999999</number>
    </property>
   </widget>
   <widget class="QLabel" name="contactsLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Contatos</string>
    </property>
   </widget>
   <widget class="QLabel" name="communicationLabel">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>470</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Raio de comunicação</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Conectar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nicknameEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Digite seu apelido</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Digite seu nome</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
